lib/config/default-config.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @fileoverview Default configuration
+ * @author Nicholas C. Zakas
+ */
+
+"use strict";
+
+//-----------------------------------------------------------------------------
+// Requirements
+//-----------------------------------------------------------------------------
+
+const Rules = require("../rules");
+
+//-----------------------------------------------------------------------------
+// Helpers
+//-----------------------------------------------------------------------------
+
+/*
+ * Because we try to delay loading rules until absolutely necessary, a proxy
+ * allows us to hook into the lazy-loading aspect of the rules map while still
+ * keeping all of the relevant configuration inside of the config array.
+ */
+const defaultRulesConfig = new Proxy({}, {
+    get(target, property) {
+        return Rules.get(property);
+    },
+
+    has(target, property) {
+        return Rules.has(property);
+    }
+});
+
+exports.defaultConfig = [
+    {
+        plugins: {
+            "@": {
```

Comment:
Is there any reason we'd want this to be a `Symbol` rather than a special string?